cmd/api/http/handlers.go

```diff
@@ -256,6 +256,49 @@ func isolateId(w http.ResponseWriter, r *http.Request) (id uuid.UUID, err error)
 	return id, nil
 }
 
+type BookResponse struct {
+	ID        uuid.UUID `json:"id"`
+	Name      string    `json:"name"`
+	Price     *float32  `json:"price"`
+	Inventory *int      `json:"inventory"`
+	Archived  bool      `json:"archived"`
+}
+
+/*Copy the fields of a book object to an http layer struct with json tags*/
+func bookToResponse(b book.Book) BookResponse {
+	return BookResponse{
+		ID:        b.ID,
+		Name:      b.Name,
+		Price:     b.Price,
+		Inventory: b.Inventory,
+		Archived:  b.Archived,
+	}
+}
+
+type PageOfBooksResponse struct {
+	PageCurrent int            `json:"page_current"`
+	PageTotal   int            `json:"page_total"`
+	PageSize    int            `json:"page_size"`
+	ItemsTotal  int            `json:"items_total"`
+	Results     []BookResponse `json:"results"`
+}
+
+/*Copy the fields of a PagedBooks object to an http layer struct with json tags*/
+func pagedBooksToResponse(page book.PagedBooks) PageOfBooksResponse {
+	jsonTagedBooks := []BookResponse{}
+	for _, book := range page.Results {
+		jsonTagedBooks = append(jsonTagedBooks, bookToResponse(book))
+	}
```

Comment:
I am not sure about the variable name here. `jsonTagedBooks` looks like a little too much, maybe use `results` or `bookResults`. This is a pretty short lived function, you don't need to have such big names.